Dockerfile

```diff
@@ -3,54 +3,45 @@
 # This Dockerfile is built as the `dimagi/commcarehq_base` image, which
 # is used for running tests.
 
-FROM python:3.9
+FROM ghcr.io/astral-sh/uv:0.5.2-python3.9-bookworm-slim
 MAINTAINER Dimagi <devops@dimagi.com>
 
 ENV PYTHONUNBUFFERED=1 \
     PYTHONUSERBASE=/vendor \
     PATH=/vendor/bin:$PATH \
-    NODE_VERSION=20.11.1
+    NODE_VERSION=20.11.1 \
+    UV_COMPILE_BYTECODE=1 \
+    UV_LINK_MODE=copy
```

Comment:
I know you are just following suit, but do you think it is worth explaining why these options should be set? Or are the docs on these options clear enough that we don't need additional documentation in our own code?

Reply:
https://github.com/dimagi/commcare-hq/pull/35375/commits/f4746de45bacfcc95f49c46402f40f78f7686ac1

Reply:
nit:
> UV_COMPILE_BYTECODE: Compile bytecode during installation.

This doesn't feel like quite what I was imagining. This explains what it does, but not why we need it right? I was imagining the comment conveying why it is needed. The comment for UV_LINK_MODE is more like what I was thinking. I recognize how nitpicky this, but I'd just assume not have this comment at all rather than what it currently is.

Reply:
Yes, you're right. I briefly considered not putting a comment on that one, but decided to add it because you specifically included that line in your comment scope. I couldn't think of a better thing to say at the time, but I could add that it makes modules load faster, which seems like a good enough explanation of why. Do you agree?